testem.js

```diff
@@ -1,11 +1,11 @@
 module.exports = {
-  disable_watching: true,
-  framework: 'mocha',
-  launch_in_ci: [
-    'Firefox'
-  ],
-  launch_in_dev: [
+  'framework': 'mocha',
+  'test_page': 'tests/index.html?hidepassed&coverage',
+  'disable_watching': true,
+  'launch_in_ci': [
```

Comment:
I much prefer to keep this as Firefox so it gets tested in both browsers. I know most devs including myself end up doing development in Chrome and running tests in Chrome so I want the CI testing against Firefox. If you really want the CI to test Chrome then please make it do both.